skimage/graph/_mcp.pyx

```diff
@@ -391,9 +391,22 @@ cdef class MCP:
         """
         pass
 
-
+    @deprecate_parameter(
+        "max_cumulative_cost",
```

Comment:
Would these eventually be useful to have, to reject paths that go beyond a given value? We can add them back later, just wondering.

Reply:
@michaelbratsch Thoughts?